jazz_deployments/index.js

```diff
@@ -544,4 +548,28 @@ function buildNowRequest(serviceDetails, config, refDeployment) {
 			reject("unable to find deployment details");
 		}
 	});
-}
\ No newline at end of file
+}
+
+const factory = {
```

Comment:
This variable name is confusing as it isn't a `factory` in traditional terms.

Reply:
Done